src/js/controller/CursorCoordinatesController.js

```diff
@@ -45,7 +46,13 @@
       html += '<div class="drawing-zoom">x' + zoom + '</div>';
     }
 
-    this.coordinatesContainer.innerHTML = this.getFrameSizeHTML_() + html;
+    this.coordinatesContainer.innerHTML = this.getFrameSizeHTML_() + html + this.getCurrentFrameIndex_();
+  };
+
+  ns.CursorCoordinatesController.prototype.getCurrentFrameIndex_ = function () {
```

Comment:
Can we just rename this method to have HTML in the name, for instance `getCurrentFrameIndexHTML_` (or shorter if you can find anything else :) _

Reply:
Done.